src/utils/articleTextSize.js

```diff
@@ -1,8 +1,6 @@
 const KEY = 'article-textsize'
-const DEFAULT_SIZE = 2
-const [MIN_SIZE, MAX_SIZE] = [0, 6]
-const SELECTORS = '.adjustable-font-size'
-const FONT_SIZE_ATTRIBUTE = 'data-font-size'
+const DEFAULT_SIZE = 3
+const [MIN_SIZE, MAX_SIZE] = [1, 7]
```

Comment:
Shifted from 0-6 to 1-7 because lists in LESS start counting from 1 (not sure why)

http://lesscss.org/functions/#list-functions-extract